waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/CloseableThriftHiveMetastoreIfaceClientFactory.java

```diff
@@ -59,7 +59,7 @@ public CloseableThriftHiveMetastoreIface newInstance(AbstractMetaStore metaStore
     }
     Map<String, String> properties = new HashMap<>();
     properties.put(ConfVars.METASTOREURIS.varname, uris);
-    HiveConfFactory confFactory = new HiveConfFactory(Collections.emptyList(), properties);
+    HiveConfFactory confFactory = new HiveConfFactory(null, properties);
```

Comment:
See my comment in `CommonBeans`: same here, we rather add the properties from the yaml for the client HiveConf's.

Reply:
Yes, ideally this shouldn't need to change.